### DISCUSSION TITLE
Ledger Nano S not connecting

### DISCUSSION THREAD

**Discussion by Ogreboh** [iotaledger/firefly#8577] [Help & Support]
Hi! I need help. I created an IOTA account on Firefly and connected with my Ledger Nano S months ago. I updated the Ledger Live software on my Nano S (IOTA, etc..) but now when I try to connect my Nano S to Firefly, it is getting hung on a "Connect your Ledger device to continue" screen. 

Ledger Live will recognize and access my Nano S but now Firefly won't (I have tried moving the USB cord to another port, made sure Ledger Live was closed, IOTA app on Nano S was open, etc..). 

Any help is greatly appreciated!

**Comment by begonaalvarezd** [comment]
Are you using the official ledger cable? Ledger does not behave very well without the official cable

**Comment by Ogreboh** [comment]
I am. I installed the apps on my desktop and was able to connect/view on that system. Firefly doesn't like my laptop anymore for some reason, it seems. Thanks for the reply as it pushed me to try a desktop setup!
